be/src/exec/hash_joiner.cpp

```diff
@@ -332,10 +359,10 @@ Status HashJoiner::reset_probe(starrocks::RuntimeState* state) {
         return Status::OK();
     }
 
-    _probe_input_chunk.reset();
-    _ht_has_remain = false;
```

Comment:
_ht_has_remain is missing, is it should be added into HashJoinProber::reset or HashJoinBuilder::reset_probe?

Reply:
_ht_has_remaining will be reset in HashJoinProber::reset 
```c++
void HashJoinProber::reset() {
    _probe_chunk->reset();
    _current_probe_has_remain = false;
}
```